masaya/HTML/02_createSection/createSection.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>createSection</title>
+</head>
+<body>
+    <h1>吾輩は猫である</h1>
+    <h2>夏目漱石</h2>
+    　<p>吾輩は猫である。名前はまだ無い。<br>
```

Comment:
```suggestion
    <p>吾輩は猫である。名前はまだ無い。<br>
```
pタグの開始にインデントがあるのが気になります！
htmlのルールとして親子関係が生じた時に一つインデントを空けるルールがあります！
例)

```

<ul>
  <li>hoge</li>
  <li>huga</li>
</ul>

```
今回の場合だと親子関係が生じていないのでインデントは不要です！
読み手に「ここに親子関係があるんだ」と誤解させてしまいます！